bridge/setu/listener/heimdall.go

```diff
@@ -32,7 +32,7 @@ func NewHeimdallListener() *HeimdallListener {
 
 // Start starts new block subscription
 func (hl *HeimdallListener) Start() error {
-	hl.Logger.Info("Starting")
+	hl.Logger.Info("Starting the heimdall listener...")
```

Comment:
Do we need this?

Reply:
how we know that heimdall listener started